modules/data.atmosphere/R/download.MACA.R

```diff
@@ -0,0 +1,145 @@
+##' Download MACA CMIP5 outputs for a single grid point using OPeNDAP and convert to CF 
+##' @name download.MACA
+##' @title download.MACA
+##' @export
+##' @param outfolder
+##' @param start_date
+##' @param end_date
+##' @param lat
+##' @param lon
+##' @param model , select which MACA model to run (options are CM3, ESM2M, ESM2G)
```

Comment:
Just added the other options, there are 20 models. Each takes ~45 seconds to run and only has 1 ensemble member, so I could even remove the ensemble member argument and just define it within the function